src/locales/en.json

```diff
@@ -488,10 +488,17 @@
     "CANCEL_BUTTON": "Cancel"
   },
   "AUTH_REQUEST_SCREEN": {
-    "TITLE": "Connection",
+    "TITLE": "Connection Request",
+    "ACCOUNT": "Account",
+    "BITCOIN_ADDRESS": "Bitcoin address",
+    "ORDINAL_ADDRESS": "Ordinal address",
```

Comment:
```suggestion
    "BITCOIN_ADDRESS": "Payments address",
    "ORDINAL_ADDRESS": "Ordinals address",
```

Reply:
@m-aboelenein I think you updated the translations for btcSelectAddressScreen, but not authenticationRequest screen

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/cc927fcd-f015-4fc4-a720-85814da3fe26)